Makefile

```diff
@@ -7,6 +7,28 @@ OPERATOR_NAME := machine-deletion-remediation
 # See github.com/operator-framework/operator-sdk/releases for the last version
 OPERATOR_SDK_VERSION ?= v1.25.1
 
+# versions at https://github.com/operator-framework/operator-registry/releases
+OPM_VERSION = v1.15.1
+
+# versions at  https://github.com/kubernetes-sigs/controller-tools/releases
+CONTROLLER_GEN_VERSION = v0.9.2
+
+# versions at https://github.com/kubernetes-sigs/kustomize/releases
+KUSTOMIZE_VERSION = v4.5.7
+# update for major version updates to KUSTOMIZE_VERSION!
+KUSTOMIZE_API_VERSION = v4
+
+ # no tagged versions
+ENVTEST_VERSION = v0.0.0-20230208013708-22718275bffe
+# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
+ENVTEST_K8S_VERSION = 1.23
+
+# See https://pkg.go.dev/golang.org/x/tools/cmd/goimports?tab=versions for the last version
+GOIMPORTS_VERSION ?= v0.6.0
+
+# See https://github.com/slintes/sort-imports/releases for the last version
+SORT_IMPORTS_VERSION = v0.2.1
+
```

Comment:
Do you refer to KUSTOMIZE and ENVTEST? In that cases I removed the line on purpose because they belong to the same "component", so I would keep them together.

I noted now that there is a whitespace at the beginning of line 21

Reply:
> I noted now that there is a whitespace at the beginning of line 21

I wasn't referring to that, but we can indeed remove that.

> In that cases I removed the line on purpose because they belong to the same "component", so I would keep them together.

I didn't notice you have grouped _KUSTOMIZE_ and _ENVTEST_ version lines. I don't have strong preference.